synthetic_phi/whole_slide_image/src/isyntax.py

```diff
@@ -0,0 +1,159 @@
+import os
+import traceback
+
+from lxml import etree as ET
+import base64
+import io
+from PIL import Image, ImageDraw
+
+def generate_image(x, y, image_text1=None, image_text2=None):
+    img = Image.new('RGB', (x, y), color=(73, 109, 137))
+
+    d = ImageDraw.Draw(img)
+    d.text((100, 100), "REMOVED DATA 1", fill=(255, 255, 0))
```

Comment:
same here and below. You can do something like: https://github.com/project-medid/project-medid/blob/main/synthetic_phi/ultrasound/src/utils.py#L80

Reply:
Done.